frontend/src/components/forms/ProfileEditForm.tsx

```diff
@@ -181,10 +185,12 @@ const ProfileEditForm: FunctionComponent<Props> = ({
           const selectValue = useMemo(() => {
             if (item.forced === "True") {
               return "forced";
-            } else if (item.hi === "True") {
-              return "hi";
-            } else {
-              return "normal";
+            } else if (item.hi === "also") {
+              return "also";
+            } else if (item.hi === "only") {
+              return "only";
+            } else if (item.hi === "never") {
+              return "never";
```

Comment:
if we narrow down the types we could simplify this as

```suggestion
            } else {
              return item.hi
```